### PR TITLE
Add OmegaT 4.2.0

### PR DESCRIPTION
**Pull request by linsui**

### CODE REVIEW

File: bucket/omegat.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            "url": "https://downloads.sourceforge.net/project/omegat/OmegaT%20-%20Latest/OmegaT%204.2.0/OmegaT_4.2.0_Beta_Windows_64_Signed.exe",
+            "hash": "sha1:52b94bc3e9f154b03302e123b339415e414790cb"
+        },
+        "32bit": {
+            "url": "https://downloads.sourceforge.net/project/omegat/OmegaT%20-%20Latest/OmegaT%204.2.0/OmegaT_4.2.0_Beta_Windows_Signed.exe",
+            "hash": "sha1:f1edf89057de2931ce3546dfe34fcded6692cc02"
+        }
+    },
+    "innosetup": true,
+    "bin": [
```
Comment (Ash258) [inline bucket/omegat.json]:
```suggestion
    "bin": "OmegaT.exe",
```

File: bucket/omegat.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        "url": "https://sourceforge.net/projects/omegat/rss?path=/OmegaT%20-%20Latest",
+        "regex": "OmegaT_([\\d._]+)_Beta"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://downloads.sourceforge.net/project/omegat/OmegaT%20-%20Latest/OmegaT%20$version/OmegaT_$version_Beta_Windows_64_Signed.exe"
+            },
+            "32bit": {
+                "url": "https://downloads.sourceforge.net/project/omegat/OmegaT%20-%20Latest/OmegaT%20$version/OmegaT_$version_Beta_Windows_Signed.exe"
```
Comment (Ash258) [inline bucket/omegat.json]: Beta?

Comment (linsui) [reply bucket/omegat.json]: > Standard
> The "Standard" version of OmegaT includes a full user manual (at least in English) and will usually have been tested for some time to eliminate any bugs. It is the recommended version for first-time users.

> Latest
> As its name suggests, the "Latest" version of Omegat is more up to date. However, the user manual may no longer be up to date (and therefore possibly confusing), and some minor bugs are possible. Nevertheless, the "Latest" versions of OmegaT are usually very stable and are recommended for any users who already have some experience with OmegaT.

The latest version is tagged as beta.

Comment (linsui) [reply bucket/omegat.json]: @Ash258 Is it OK?

**Pull request merged**